### ISSUE TITLE
[HC] Menustrips' child control textbox borders are not visible

### ISSUE THREAD

**Issue by Nora-Zhou01** [dotnet/winforms#7740]
### .NET version

8.0.100-alpha.1.22451.15

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, it is not a regression issue

### Issue description

Cu**rrent Behavior:**
Menustrips' child control textbox borders are not visible
HC Aquatic theme:
![A6](https://user-images.githubusercontent.com/104609169/188589900-c1403e7c-40a4-4861-9c24-2d72744781f0.png)

HC Desert theme:
![A2](https://user-images.githubusercontent.com/104609169/188589977-85983551-4492-4968-a120-1f9d6d4e2892.png)

HC Dusk theme:
![A3](https://user-images.githubusercontent.com/104609169/188590019-bd145af5-e015-4a5f-8587-32e6e74b4dc7.png)

HC Night Sky theme:
![A4](https://user-images.githubusercontent.com/104609169/188590061-826f1679-9a22-4701-838c-6dd7328102f6.png)

E**xpected Behavior:**
Menustrips' child control textbox borders can be seen
![1](https://user-images.githubusercontent.com/104609169/188532860-eb07a017-890c-4009-9156-97b772ca7875.png)

### Steps to reproduce

1. Turn on High Contrast theme in Windows settings
2. Create a WinForms .NET core project
3. Add MenuStrip control
4. Add child controls MenuItem, TextBox, ComboBox respectively
5. Ctrl+Shift+A add a Application Manifest File and uncomment windows 10
6. Build and run

**More Info:**

1. This issue is repro on all HC themes.
2. This issue is repro on framework project.

**Comment by merriemcgaw** [comment]
We should do a textbox border in HC modes, just using the regular system color for control border, IMO.

**Comment by merriemcgaw** [comment]
Experience in this particular application in standard theme is almost as bad as HC. 
@Nora-Zhou01 what happens here for Win10 HC themes 
@merriemcgaw to work with Scott to figure out whether or not we need to force the border in HC mode and what the priority might be.

**Comment by scottaohara** [comment]
@merriemcgaw yeh this definitely looks to be an issue.  i'd have no idea there was a form field there in the high contrast modes.

**Comment by Nora-Zhou01** [comment]
@merriemcgaw the text in the textbox can be displayed and the border can be shown when the cursor is placed on the textbox.
Win10 high contrast themes:
HC #1: 
![H1](https://user-images.githubusercontent.com/104609169/189089527-3eed5358-8a38-4177-8c80-a41f44760b69.png)

HC #2: 
![H2](https://user-images.githubusercontent.com/104609169/189089544-3a43e0b3-b705-4b68-9ae8-346e8dbade1a.png)

HC Black: 
![H3](https://user-images.githubusercontent.com/104609169/189089558-75383ce9-20e0-4abd-a704-f9f975462814.png)

HC White: 
![H4](https://user-images.githubusercontent.com/104609169/189089590-d8702ee4-df7d-481f-adc9-4e11f7daa75b.png)

**Comment by merriemcgaw** [comment]
@vladimir-krestov / @dmitrii-drobotov can you all get this fixed for 7.0? I think if it's not risky we could take it in RC2.

**Comment by John-Qiao** [comment]
Verified with 8.0.100-alpha.1.22469.15 from main branch, this issue has been fixed that the textbox border is visible in MenuStrip now.
 
![TestResult2](https://user-images.githubusercontent.com/45864985/191185283-1c0c8bb1-1800-4578-bf1d-7bca9a50ca17.png)

**Comment by Olina-Zhang** [comment]
> @vladimir-krestov / @dmitrii-drobotov can you all get this fixed for 7.0? I think if it's not risky we could take it in RC2.

Do you have plan to service the fixing to .Net 7.0? I didn't see the corresponding PR, just for main.

**Comment by Tanya-Solyanik** [comment]
Thank you, @Olina-Zhang! 
We are in M2 approval mode for .NET 7 GA. Branch is release/7.0

**Comment by Nora-Zhou01** [comment]
Verified on latest .NET 7 build .NET 7.0.100-rtm.22478.24,this issue has been fixed that the textbox border is visible in MenuStrip now,the results are consistent with the above.

**Comment by Cassie-Li01** [comment]
Verified this issue on .NET 7.0.100 GA build, issue was fixed. The screenshot as same as above.
